pkg/pool/operations/import.go

```diff
@@ -53,14 +56,18 @@ func (oc *OperationsConfig) Import(cspi *cstor.CStorPoolInstance) (bool, error)
 	klog.Infof("Importing pool %s %s", string(cspi.GetUID()), PoolName())
```

Comment:
Should we need to have a log saying that renaming the pool? It will help us to understand user is renaming existing pool.

Reply:
Done